src/main/java/net/javadiscord/javabot/systems/qotw/QOTWJob.java

```diff
@@ -72,7 +72,7 @@ private MessageEmbed buildQuestionEmbed(QOTWQuestion question) {
 		var checkTime = OffsetDateTime.now().plusDays(6).withHour(22).withMinute(0).withSecond(0);
 		return new EmbedBuilder()
 				.setTitle("Question of the Week #" + question.getQuestionNumber())
-				.setDescription(String.format("%s\n\nClick the button below to submit your answer.\nYour answers will be checked by <t:%d:F>",
+				.setDescription(String.format("%s%n%nClick the button below to submit your answer.%nYour answers will be checked by <t:%d:F>",
```

Comment:
SonarLint suggested to use them, lol.

Reply:
SonarLint suggests the use of `%n` instead of `\n` because `%n` inserts platform-specific line breaks which is a good idea when  writing to files. However, this doesn't make sense for transmitting text to the discord API.

Reply:
Yeah, I know, does it make any difference, tho?

Reply:
Okay, yeah. Just removed them and disabled the rule